gym/utils/play.py

```diff
@@ -261,9 +256,9 @@ def play(
         else:
             action = key_code_to_action.get(tuple(sorted(game.pressed_keys)), noop)
             prev_obs = obs
-            obs, rew, done, info = env.step(action)
+            obs, rew, terminated, truncated, info = env.step(action)
```

Comment:
need to add `done = termianted or truncated` after otherwise `done` is never updated